docs/content/configuration/csi_ug_config_create_replication.md

```diff
@@ -0,0 +1,46 @@
+# Creating a VolumeReplication
+
+Create a VolumeReplication yaml file to replicate a specific PersistentVolumeClaim (PVC).
+
+VolumeReplicationClass needs to be present before a VolumeSnapshot can be created. For more information, see [Creating a VolumeReplicationClass](csi_ug_config_create_vol_replicationclass.md).
+
+**Note:** Replication is also known as mirroring in many storage systems.
+
+When replicating a volume, be sure to follow all of the replication configurations, found in [Compatibility and requirements](../installation/csi_ug_requirements.md) before snapshot creation.
```

Comment:
`snapshot` has nothing to do with replication

Reply:
DONE
@oriyarde  I understand - I copied the snapshot file and worked off of that and just forgot to change it over.

Reply:
I don't see any change here yet

Reply:
@oriyarde  Please check now. I might have not pushed the changes fully earlier.